sampleapp/src/main/java/com/dlazaro66/sampleapp/MainActivity.java

```diff
@@ -10,33 +14,68 @@
 
 public class MainActivity extends Activity {
 
+    private LinearGradient gradientOne;
+    private LinearGradient gradientTwo;
+    private int gradientAcitivtyPosition;
+    private int gradientSwitch = 1;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-        WheelIndicatorView wheelIndicatorView = (WheelIndicatorView) findViewById(R.id.wheel_indicator_view);
+        final WheelIndicatorView wheelIndicatorView = (WheelIndicatorView) findViewById(R.id.wheel_indicator_view);
 
         //WheelIndicatorView wheelIndicatorView = new WheelIndicatorView(this);
 
         // dummy data
         float dailyKmsTarget = 4.0f; // 4.0Km is the user target, for example
         float totalKmsDone = 3.0f; // User has done 3 Km
-        int percentageOfExerciseDone = (int) (totalKmsDone/dailyKmsTarget * 100); //
+        int percentageOfExerciseDone = (int) (totalKmsDone / dailyKmsTarget * 100); //
 
         wheelIndicatorView.setFilledPercent(percentageOfExerciseDone);
 
-        WheelIndicatorItem bikeActivityIndicatorItem = new WheelIndicatorItem(1.8f, Color.parseColor("#ff9000"));
```

Comment:
I would love to maintain the previous example instead of replacing it with the new one you have done. Please, can you write a new example with the two WheelIndicatorItem creation modes? For example, create a new WheelIndicatorView below the existing one.
Thank you a lot!